extensions/spark/kyuubi-extension-spark-3-4/src/main/scala/org/apache/kyuubi/sql/InsertShuffleNodeBeforeJoin.scala

```diff
@@ -49,7 +49,7 @@ object InsertShuffleNodeBeforeJoin extends Rule[SparkPlan] {
     }
   }
 
-  // Since spark 3.3, insertShuffleBeforeJoin shouldn't be applied if join is skewed.
+  // SPARK-33832 (Spark 3.3) already handles skew join cases
```

Comment:
As https://github.com/apache/kyuubi/pull/6260#issuecomment-2041317569 commented, should we revert this change?

Reply:
yes, the comment is incorrect then. but seems SPARK-46170 recovers the skew join cases